src/index.js

```diff
@@ -17,21 +18,29 @@ class App extends React.Component {
   }
 
   componentDidMount() {
+    const uniqueOwnerId = Math.random().toString(36).substring(2, 15)
```

Comment:
Only need to perform this if the cookie load is undefined, could be moved within the if statement. Also a bit funky the randomisation, could use something like: https://www.npmjs.com/package/uniqid